lib/generators/rspec/templates/query_spec.rb.erb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+<% module_namespacing do -%>
+RSpec.describe <%= class_name %>Query do
+  subject(:query) { described_class.new }
+
+  describe '#all' do
+    let!(:<%= table_name %>) { create_list(:<%= factory %>, 3) }
```

Comment:
so this would mean we have a dependency on FactoryBot?

Reply:
🤔 I didn't think about it. Maybe it's not a good idea to add the default test example. I should delete it, shouldn't I?

Reply:
Yes, I agree maybe no actual tests it's better (but still creating the file such as rails/rspec do with models)